src/patterns/find-a-support-centre/examples/results/_support-centre.scss

```diff
@@ -0,0 +1,42 @@
+/* --------------------------------
+
+Title:
+Support centre
+
+Description:
+Support centre listing
+
+-------------------------------- */
+
+.support-centre {
+  margin-bottom: 0;
+  padding-left: 0;
+}
+
+.support-centre__item {
+  display: block;
+  margin-bottom: 0;
+
+  p:last-child {
+    margin-bottom: 0;
+  }
+}
+
+.support-centre__item:not(:first-child) {
+  border-top: 1px solid #d8d8d8;
+  padding-top: 32px;
+}
+
+.support-centre__item:not(:last-child) {
+  padding-bottom: 32px;
+}
+
+.support-centre__distance {
+  background-color: rgba(#00a3a6, 0.2);
```

Comment:
Can this colour be added to the style as a new colour

Reply:
This is a Census specific tint of a colour and does not seem to be available. Perhaps we can have Census specific colours if we have not already got them. They don’t seem to be available from the documentation.

Reply:
We do have Census specific colour variables. If you look in `scss/settings/` you'll find a `census.scss` file which has all the census specific variables.

Reply:
I'm not sure we need this stylesheet? It looks like many of these styles could be sorted with utility classes. 

For any styles we do need (the distance element) I think we could possible drop the `support-centre` bit and keep it generic.

Reply:
I have removed anything specific to support-centre and used utility classes as you suggested. I have also added the supporting colour for Census (teal) and a tint of 20%. This needs to be revisited at some point.